app/app/crud/crud_student.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict, Optional, Union
+
+from sqlalchemy.orm import Session
+
+from app.crud.base import CRUDBase
+from app.models import Student
+from app.schemas import StudentCreate, StudentUpdate
+
+
+class CRUDStudent(CRUDBase[Student, StudentCreate, StudentUpdate]):
+    def get(self, db: Session, id: int) -> Optional[Student]:
```

Comment:
maybe add a `get_students_by_term` either here or in `crud_term`

Reply:
Adding in term would be better, since this actually term ka feature

Reply:
Hmm okay